pkg/asset/agent/image/ignition.go

```diff
@@ -360,6 +373,53 @@ func addExtraManifests(config *igntypes.Config, extraManifests *manifests.ExtraM
 	}
 }
 
+func getOSImagesInfo() (*models.OsImage, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+	defer cancel()
+
+	st, err := rhcos.FetchCoreOSBuild(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Architecture currently hard-coded to x86_64
+	cpurArch := archName
+	osImage := &models.OsImage{
+		CPUArchitecture: &cpurArch,
+	}
+
+	ver, err := version.Version()
```

Comment:
for a moment, it looked like `var`. 
`version, err := version.Version()`

Reply:
fair point :)